### PR TITLE
Feature/amp conf

### PR DESCRIPTION
**Pull request by keevkeev**
new ampconf layout

### CODE REVIEW

File: frontend/scss/components/molecules/agenda-item.scss [hunk 1]
```diff
@@ -15,56 +15,122 @@
...
+      border-radius: 50%;
+      align-self: flex-start;
+      margin-right: 10px;
+      margin-left: -60px;
+    }
+
+    &.multiple {
+      position: absolute;
+
+      &:nth-child(1) {
```
Comment (matthiasrohmer) [inline frontend/scss/components/molecules/agenda-item.scss]: Having that many `:nth-child` selectors doesn't seem like the most intuitive solution 😅

File: frontend/scss/components/molecules/agenda-item.scss [hunk 2]
```diff
@@ -15,56 +15,122 @@
...
     width: 80%;
-    margin-right: auto;
-    padding: 7px 0 7px 25px;
     border-left: 2px solid color('blue-ribbon');
-    position: relative;
-    display: flex;
-    align-items: center;
-    color: color('charade');
-
+    padding: 13.5px 0 13.5px 25px;
```
Comment (matthiasrohmer) [inline frontend/scss/components/molecules/agenda-item.scss]: Please don't use half-pixel values.

File: frontend/scss/components/organisms/agenda.scss [hunk 3]
```diff
@@ -33,6 +33,39 @@
...
+      flex-grow: 1;
+      outline: none;
+      color: fade_out(color('limed-spruce'), 0.57);
+      background: color('athens-gray');
+
+      &+.#{organism('agenda')} {
+        display: none;
+      }
+
+      &[option][selected] {
```
Comment (matthiasrohmer) [inline frontend/scss/components/organisms/agenda.scss]: Does that selector need to be double specific by using `[option]` and `[selected]` or would the later one be sufficient?

File: frontend/scss/components/organisms/stage-ads.scss [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+  @media (min-width: 768px) {
+    margin-left: 64px;
+    border-bottom-left-radius: 6.250em;
+    -webkit-clip-path: polygon(0 0, 100% 0, 100% calc(100% - 3em), 6.250em 100%, 0% 100%);
+    clip-path: polygon(0 0, 100% 0, 100% calc(100% - 3em), 6.250em 100%, 0% 100%);
+  }
+
+
+  div.#{utility('container-fluid')} {
```
Comment (matthiasrohmer) [inline frontend/scss/components/organisms/stage-ads.scss]: Does that selector need to be that specific by prepending `div`?

File: frontend/scss/components/organisms/agenda.scss (hunk 3 above)
Comment (keevkeev) [reply frontend/scss/components/organisms/agenda.scss]: amp-selector adds some default stylings to selected options using this double selector and without the double selector some of our styling properties are overridden (e.g. border: 'none')

File: frontend/scss/components/organisms/stage-ads.scss (hunk 4 above)
Comment (keevkeev) [reply frontend/scss/components/organisms/stage-ads.scss]: all stage organisms are wrapped in a <section> tag that utilizes the "ap--container-fluid" class, figured using this selector to style the individual div was less css than creating a whole new utility class with the same properties as "ap--container-fluid" with the exception for a different padding

File: frontend/scss/components/molecules/agenda-item.scss (hunk 2 above)
Comment (keevkeev) [reply frontend/scss/components/molecules/agenda-item.scss]: sorry about that

**Pull request merged**